app/packages/operators/src/components/OperatorExecutionTrigger/index.tsx

```diff
@@ -54,40 +46,18 @@ export const OperatorExecutionTrigger = ({
   onClick?: () => void;
   onSuccess?: ExecutionCallback;
   onError?: ExecutionErrorCallback;
+  onCancel?: () => void;
```

Comment:
Looks like `onCancel` is not being used in this component? Except for maybe `Box`?